### ISSUE TITLE
I seny Ton coins to my wallet adress without comment

### ISSUE THREAD

**Issue by RMTRADE** [toncenter/ton-wallet#116]
Hello,

I have sent 16,9 ton coins to my wallet adress without writing anything but just my wallet adress without a comment optional (Trans)

Could you please help me where is my ton coins and how can I reach them back.

here is my wallet adress

transaction time is
![Sem título-1](https://user-images.githubusercontent.com/102019922/159224547-d50ce46a-4190-4d36-aa04-cb55a293d277.jpg)
14:05
16,9 ton coins
20/03/2022

EQDjS0Cia1EAmWZi-6uPMp5ZEGaPMz1CYsh_gfAuxFDGeyrW

**Comment by 78arda** [comment]
Have you got any answer to your question? Are there anyone replying to these issues?

**Comment by CriptoRG** [comment]
Same problem here.

**Comment by marattulepov** [comment]
> Та же проблема здесь.



> Жду ответа по проблеме.
> 
> 78arda - Это единственный сайт, который я нашел: https://tonscan.org/

hi did you solve your problem? Is it worth it to wait for the problem to be solved?

**Comment by CriptoRG** [comment]
No, i did't.Enviado desde mi Huawei de Claro.-------- Original message --------From: marattulepov ***@***.***>Date: Sat, May 7, 2022, 6:08 AMTo: toncenter/ton-wallet ***@***.***>Cc: CriptoRG ***@***.***>, Comment ***@***.***>Subject: Re: [toncenter/ton-wallet] I seny Ton coins to my wallet adress without comment (Issue #116)



Та же проблема здесь.





Жду ответа по проблеме.

78arda - Это единственный сайт, который я нашел: https://tonscan.org/



hi did you solve your problem? Is it worth it to wait for the problem to be solved?
